app/src/main/res/values/strings.xml

```diff
@@ -208,85 +208,35 @@
-        \n
-        \n
-
-How We Collect Data:
-        \n
-
-Mobile Applications:
-        \n
-
-Photos: time-stamped, Geo-tagged.
-
-        \n
-User entered information: name, email address, organization name, phone number, field notes.
-        \n
-
-App settings for users
-        \n
-
-Application analytics: software crash reports, phone make and model.
-        \n
-
-Website Application: Database entries, general analytics.
-        \n
-
-Donor supplied information
-        \n
-        \n
-
-User Data We Collect:
-        \n
-
-Information relating to individual users or organizations will not be disclosed without prior consent. We are handling all user related data (full name, phone numbers, email) securely using modern cryptography. Planter’s full names and contact information will never be shared without prior consent.
-        \n
-
-Personal Information: On our tree tracker app we require a name, and email, and optionally a phone number and an organization name. All personal data is stored in a secure database.
-        \n
-
-Mobile App users: Our mobile apps collect the following personal information: User entered information: name, email address, organization name, phone number.
-        \n
-
-Donor information: When you make a donation we collect information that is used solely for accounting, tax reporting and donation processing purposes. This information is guarded with utmost care and will not be sold, shared or traded with any other entity. We do abide by IRS requirement to disclose limited information that is mandated to be available on public taxes records. Donor information may also be disclosed at the donor\'s request. We collect the following donor information: name, address, amount of donation, any other personal information given to us.
-        \n
-
-User Photos:
-        \n
-
-GreenStand’s tree tracking program runs on the collection of time-stamped, geotagged photos. This information is stored in a secure database from which it is displayed in the form of a tree map. We also use this information to generate statistics related to tree planting. We are only interested in tree-related photos, please don’t send us anything else. With the proper use of this system, all photo taken will relate to environmental data and GreenStand does not consider or treat uploaded photos as “user data.” By using our mobile application you are agreeing to allow us to use your photos.
-        \n
-        \n
-
-Changes to the Policy
-        \n
-
-It is our intent to hold the principles of this policy, but we do understand there may be a need to change it from time to time. Changes will be posted on our website as soon as possible.
-        \n
-        \n
-
-© 2019 GreenStand (registered 501(c)3 nonprofit. EIN: 47-3150283 ). Info@greenstand.org
-        \n
-Privacy Policy
-
+	    <![CDATA[
```

Comment:
Please undo this HTML change. This should be done in a different PR. This requires code changes.